source/tests/universal/common/cases/loss/utils.py

```
@@ -0,0 +1,79 @@
+    GLOBAL_SEED,
+)
+
+
+class LossTestCase:
+    """Common test case for loss function."""
+
+    def setUp(self):
+        pass
+
+    def test_label_keys(self):
+        module = self.forward_wrapper(self.module)
```

Ruff

Local variable `module` is assigned to but never used

Remove assignment to unused variable `module`

(F841)